code/modules/SCP/SCPs/SCP-294.dm

```diff
@@ -35,40 +30,41 @@ GLOBAL_LIST_EMPTY(scp294_reagents)
 
 /obj/machinery/scp294/attack_hand(mob/user)
 
-	if((last_use + 3 SECONDS) > world.time)
-		visible_message("<span class='notice'>[src] displays NOT READY message.</span>")
+	if((last_use + 20 SECONDS) > world.time)
```

Comment:
20 SECONDS could also be made into a use_cooldown var, but that is not a strict necessity